src/layouts/BaseLayout/BaseLayout.jsx

```diff
@@ -39,20 +64,15 @@ function BaseLayout({ children }) {
       ) : (
         <Suspense fallback={null}>
           <LimitedHeader />
+          <InstallationHelpBanner />
```

Comment:
You'll have to update this here as well or else it may also show on the different non-full experiences.